routes/middleware.js

```diff
@@ -8,8 +8,13 @@ exports.initLocals = (req, res, next) => {
 
   const locals = res.locals;
   locals.user = req.user;
+  const colors = require('./setup/colors.js');
+  const route = req.url.split('/')[1];
+  locals.color = Object.keys(colors).indexOf(route) > -1
```

Comment:
Okay this is a bit weird
```js
Object.keys(colors).indexOf(route) > -1 ?
```
is equivalent to
```js
colors[route] ?
```
anyway, with new names we come out with
```js
locals.color = viewColors[viewName] || viewColors.default
```
Which actually makes sense.. ?
or
```js
const viewColor = viewColors[viewName]
locals.color = viewColor !== indefined ? viewColor : viewColors.default
```
kinda more robust (empty string is falsy)